### PR TITLE
PerviousClassesFolder

### PR DESCRIPTION
**Pull request by Kumaravinash9**
PerviousClass Folder

### CODE REVIEW

File: RecorderdClasses/pre.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
```
Comment (subodhk01) [inline RecorderdClasses/pre.html]: should have used the index.html that was on the base directory

**Review by subodhk01 — CHANGES_REQUESTED** [review]
- Component is not responsive, that is very necessary.
- The component is not exactly how it was on the wireframe.

also you could have used https://flickity.metafizzy.co/ , its a really great library and can easily be made responsive and also try to make the changes on a different branch on the original repo.

**Comment by Kumaravinash9** [comment]
Now,it's responsive .so have a look

**Review by subodhk01 — CHANGES_REQUESTED** [review]
Still not responsive
Screenshot 2020-06-26 at 12 33 55

**Comment by subodhk01** [comment]
Try using https://flickity.metafizzy.co/ if you can

**Comment by subodhk01** [comment]
Merged to #3

**Pull request closed without merging**